external/Padua/padua.cpp

```diff
@@ -0,0 +1,1866 @@
+# include <cmath>
```

Comment:
see comment above for ChebTools and external libraries.

Reply:
Added the link to the origin of the code.
The authors have all the the other information (credit, license) in line 32-73.